code/game/objects/items/decorations.dm

```diff
@@ -0,0 +1,297 @@
+	icon_state = "decoration_pot_o_gold"
+
+/obj/item/decorations/sticky_decorations/flammable/leprechaun_hat
+	name = "paper leprechaun hat"
+	desc = "A paper leprechaun hat. If you find the paper leprechaun that dropped this they might give you their pot of paper gold!"
+	icon_state = "decoration_leprechaun_hat"
+
+//Easter
+
+/obj/item/decorations/sticky_decorations/flammable/easter_bunny
+	name = "paper Easter bunny"
+	desc = "A paper Easter bunny. Help him find his lost eggs!"
+	icon_state = "decoration_easter_bunny"
+
+/obj/item/decorations/sticky_decorations/flammable/easter_egg_blue
+	name = "paper Easter egg"
+	desc = "A paper Easter egg. If the chef won't let us use their eggs, then this will have to do."
+	icon_state = "decoration_easter_egg_blue"
+
+/obj/item/decorations/sticky_decorations/flammable/easter_egg_yellow
+	name = "paper Easter egg"
+	desc = "A paper Easter egg. If the chef won't let us use their eggs, then this will have to do."
+	icon_state = "decoration_easter_egg_yellow"
+
+/obj/item/decorations/sticky_decorations/flammable/easter_egg_red
+	name = "paper Easter egg"
+	desc = "A paper Easter egg. If the chef won't let us use their eggs, then this will have to do."
+	icon_state = "decoration_easter_egg_red"
+
+/obj/item/decorations/sticky_decorations/flammable/easter_egg_purple
+	name = "paper Easter egg"
+	desc = "A paper Easter egg. If the chef won't let us use their eggs, then this will have to do."
+	icon_state = "decoration_easter_egg_purple"
+
+/obj/item/decorations/sticky_decorations/flammable/easter_egg_orange
+	name = "paper Easter egg"
+	desc = "A paper Easter egg. If the chef won't let us use their eggs, then this will have to do."
+	icon_state = "decoration_easter_egg_orange"
+
+
+
+
+///////
+//Decorative structures
+///////
+
+
+/obj/structure/decorative_structures
+	icon = 'icons/obj/decorations.dmi'
+	icon_state = ""
+	density = 1
+	anchored = 0
+	max_integrity = 100
+
+/obj/structure/decorative_structures/metal
+	flags = CONDUCT
+
+/obj/structure/decorative_structures/metal/metal_angel_statue
+	name = "metal angel statue"
+	desc = "You feel a holy presence looking back at you."
+	icon_state = "metal_angel_statue"
+
+/obj/structure/decorative_structures/metal/golden_disk_statue
+	name = "golden disk statue"
+	desc = "You aren't sure what the runes say around the large plasma crystal."
+	icon_state = "golden_disk_statue"
+
+/obj/structure/decorative_structures/metal/sun_statue
+	name = "sun statue"
+	desc = "You wonder if you could be so grossly incandescent."
+	icon_state = "sun_statue"
+
+/obj/structure/decorative_structures/metal/tesla_statue
+	name = "tesla statue"
+	desc = "Lady Tesla, a powerful and dangerous mistress."
+	icon_state = "tesla_statue"
+
+/obj/structure/decorative_structures/metal/moon_statue
+	name = "moon statue"
+	desc = "Expect a lot of Vulps to howl around this thing."
+	icon_state = "moon_statue"
+
+/obj/structure/decorative_structures/metal/tesla_monument
+	name = "tesla statue"
```

Comment:
The name is tesla statue instead of tesla monument, there is another thing named tesla statue so this will probably confuse people